### ISSUE TITLE
Elements outside map

### ISSUE THREAD

**Issue by josezy** [Simon-the-Shark/django-mapbox-location-field#2]
Hey, nice work so far.

I just gave it a try but im only using the *widget*

Works good but marker and user_location render outside the map

![image](https://user-images.githubusercontent.com/1992441/60700098-3020b180-9ebc-11e9-879b-b70ed6305262.png)

or am I maybe doing something wrong?
my setup is like this
```python
from mapbox_location_field.widgets import MapInput


class MapGeopoint(MapInput):

    @property
    def media(self):
        js = (
            "js/jquery-3.2.1.min.js",
            'https://api.tiles.mapbox.com/mapbox-gl-js/v1.1.0/mapbox-gl.js',
            'https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-geocoder/v4.4.0/mapbox-gl-geocoder.min.js',
            "mapbox_location_field/js/map_input.js",
        )
        css = {
            "all": (
                "mapbox_location_field/css/map_input.css",
                'https://api.tiles.mapbox.com/mapbox-gl-js/v1.1.0/mapbox-gl.css',
                'https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-geocoder/v4.4.0/mapbox-gl-geocoder.css',
            )
        }

        return forms.Media(js=js, css=css)
```
and
```python
from custom_widgets import MapGeopoint

class MyForm(forms.ModelForm):
    class Meta:
        model = MyModel
        fields = (geopoint,)
        widgets = {'geopoint: MapGeopoint}
```

**Comment by Simon-the-Shark** [comment]
Hi, thanks for info. That's definitely something I have to fixed.

**Comment by Simon-the-Shark** [comment]
Hi there. Thank you again for openning this issue. I fixed this bug and latest version v0.2.2 does not render markers outside the map anymore. It is available at [pipy](https://pypi.org/project/django-mapbox-location-field/0.2.2/), so you can update it with `pip install --upgrade django-mapbox-location-field`
